packages/nuxt/src/module.ts

```diff
@@ -71,6 +72,21 @@ export default defineNuxtModule<ModuleOptions>({
       })
     })
 
+    // add scalar tab to DevTools
+    if (_nuxt.options.dev && _options.devtools) {
+      _nuxt.hook('devtools:customTabs', (tabs) => {
+        tabs.push({
+          name: 'scalar',
+          title: 'Scalar',
+          icon: 'https://scalar.com/favicon.svg',
```

Comment:
just realized the logo would disappear in light mode	
<img width="266" alt="Screenshot 2024-04-22 at 1 42 11 PM" src="https://github.com/scalar/scalar/assets/38922203/bc4a7f16-63a7-4a6f-bc9b-149eecdb52fa">